SimPEG/data_misfit.py

```diff
@@ -8,12 +8,28 @@
 
 
 class BaseDataMisfit(L2ObjectiveFunction):
-    """
-    BaseDataMisfit
+    r"""Base data misfit class.
+
+    Inherit this class to build your own data misfit function. The ``BaseDataMisfit``
+    class inherits the :py:class:`SimPEG.objective_function.L2ObjectiveFunction`.
+    And as a result, it is limited to building data misfit functions of the form:
 
-    .. note::
-        You should inherit from this class to create your own data misfit
-        term.
+    .. math::
+        \phi_d (\mathbf{m}) = \frac{1}{2} \| \mathbf{W} f(\mathbf{m}) \|_2^2
+
+    where :math:`\mathbf{m}` is the model vector, :math:`mathbf{W}` is a linear weighting
```

Comment:
Add missing backslash:
```suggestion
    where :math:`\mathbf{m}` is the model vector, :math:`\mathbf{W}` is a linear weighting
```